backend/src/schema/resolvers/donations.spec.js

```diff
@@ -9,9 +9,10 @@ const instance = getNeode()
 const driver = getDriver()
 
 const updateDonationsMutation = gql`
-  mutation ($goal: Int, $progress: Int) {
-    UpdateDonations(goal: $goal, progress: $progress) {
+  mutation ($showDonations: Boolean, $goal: Int, $progress: Int) {
+    UpdateDonations(showDonations: $showDonations, goal: $goal, progress: $progress) {
```

Comment:
I cannot understand why you define a quer in the test - this seems to be a hack?

Reply:
Du würdest den Query importieren? @ulfgebhardt

Reply:
What is about this one @ulfgebhardt ?